src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -659,6 +659,10 @@ typedef struct {
     // If unset GENERATED_CERTIFICATE_BITS will be used
     INT32 generatedCertificateBits;
 
+    // generateRSACertificate controls if an ECDSA or RSA certificate is generated. By default we generate an ECDSA
+    // certificate but some platforms may not support them.
+    BOOL generateRSACertificate;
```

Comment:
Not sure if there is a standard that talks about which curve and what bit-ness to use

Reply:
```
All Implementations MUST support DTLS 1.2 with the
   TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256 cipher suite and the P-256
   curve [FIPS186].  Earlier drafts of this specification required DTLS
   1.0 with the cipher suite TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA, and
   at the time of this writing some implementations do not support DTLS
   1.2; endpoints which support only DTLS 1.2 might encounter
   interoperability issues.  The DTLS-SRTP protection profile
   SRTP_AES128_CM_HMAC_SHA1_80 MUST be supported for SRTP.
   Implementations MUST favor cipher suites which support (Perfect
   Forward Secrecy) PFS over non-PFS cipher suites and SHOULD favor AEAD
   over non-AEAD cipher suites.
```

Reply:
Cool. MINOR: Could you include the link in the comment?